sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/DigitalTwinsSendComponentTelemetryOptions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <inheritdoc />
+    [CodeGenModel("DigitalTwinsSendComponentTelemetryOptions")]
+    public partial class DigitalTwinsSendComponentTelemetryOptions
```

Comment:
we have called this API `PublishComponenetTelemetry` shouldn't we rename the option to match the API name?

Reply:
I want to have a broader discussion about these options objects at the scope of Java vs .NET, and at the per SDK level. Its tricky because .NET can rename these to be prettier, but Java can't, so it becomes a question of consistency between languages vs quality per language. Stay tuned on this one.

Reply:
Oh, also add in the fact that Java vs .NET have different verb preferences when doing things like listing. We'll have plenty to discuss here as a team

Reply:
I think the verb preference in this case is up to the ADT service.